### PR TITLE
Miniupnpdtweaks

### PR DESCRIPTION
**Pull request by ldir-EDB0**
Maintainer: me 

Description:  A couple of tweaks, build size optimisation & change of leasefile location.

Tested on ar71xx Archer C7 v2

### CODE REVIEW

**Comment by diizzyy** [comment]
Nice!

File: net/miniupnpd/Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=miniupnpd
 PKG_VERSION:=2.0.20180503
-PKG_RELEASE:=1
+PKG_RELEASE:=3
```
Comment (pprindeville) [inline net/miniupnpd/Makefile]: Umm.... what happened to `2`?

Comment (speadup) [reply net/miniupnpd/Makefile]: 2 had been used in first commit

Comment (ldir-EDB0) [reply net/miniupnpd/Makefile]: There were 2 commits in the 1 PR, with makefile version bumps in both, to be applied in sequence.  The idea being to be able to revert the individual changes if needed.

**Pull request merged**